src/ucp/api/ucp.h

```diff
@@ -1243,6 +1261,46 @@ ucs_status_t ucp_worker_signal(ucp_worker_h worker);
 
 
 /**
+ * @ingroup UCP_WORKER
+ * @brief Accept connections on a local address of the worker object.
+ *
+ * This routine binds the worker object to a sock_addr which is set by the user.
+ * The worker will listen to incoming connection requests and upon receiving such
+ * a request from the remote peer, an endpoint to it will be created.
+ * The user's call-back will be invoked once the endpoint is created.
+ *
+ * @param [in]  worker           Worker object that is associated with the
+ *                               sock_addr object.
+ * @param [in]  sock_addr        Address to bind to.
+ * @param [in]  addr_len         Length of the address.
+ * @param [in]  cb               Function callback upon created connections.
+ * @param [in]  ctx              User-provided context for calls to the callback.
+ * @param [out] bind_ctx         A handler to the listener, to be used later for
+ *                               unbinding.
+ *
+ * @return Error code as defined by @ref ucs_status_t
+ */
+ucs_status_t ucp_worker_bind(ucp_worker_h worker,
```

Comment:
i think we discussed about introducing new listener object to "server", so we could have more than one listener per worker